### ISSUE TITLE
[BUG] Audio input & Output is not working 

### ISSUE THREAD

**Issue by PSGitHubUser1** [mylinuxforwork/dotfiles#38]
The issue started when I reinstalled the dotfiles.

I can't listen any sound.. doesn't matter I am using YouTube, VLC or using Discord.

My headphones' mic & my headphones' speaker aren't working at all. I reinstalled PulseAudio, Pipewire too.

**Comment by mylinuxforwork** [comment]
Interesting. No issues on my systems. What happens if you click on the volume module in waybar?

**Comment by Ahorts** [comment]
Also, does your device have a mute button or something of that sort? I sometimes experience this when I boot into arch. The speaker seems to be disabled until I press the mute key and unmute it again. I might test this on KDE to confirm if its from hyprland or not.

**Comment by Ahorts** [comment]
I have the same thing happening in KDE too. If you reinstalled Arch I doubt this issue is from the dotfiles. I mostly use bluetooth on my headphone and that works fine. It's the speakers where I have to mute and unmute to get the audio working. 

I might have to look into this further

**Comment by mylinuxforwork** [comment]
Thanks for looking into it further. I also doubt that this comes from the dotfiles if the same happens on KDE. But let’s keep the issue open for few days more.

**Comment by PSGitHubUser1** [comment]
> Do you have another Desktop Environment?

Hyprland, Arch

**Comment by PSGitHubUser1** [comment]
> Interesting. No issues on my systems. What happens if you click on the volume module in waybar?

There is no module of waybar, I don't think there was even a volume button on waybar, At my first install too

**Comment by PSGitHubUser1** [comment]
> Also, does your device have a mute button or something of that sort? I sometimes experience this when I boot into arch. The speaker seems to be disabled until I press the mute key and unmute it again. I might test this on KDE to confirm if its from hyprland or not.

Mine worked before, but after reinstalling arch, mic & speakers are not working.

And I think this is an Arch issue.. because as far as I remember, mics & sounds don't work on KDE 6 too

**Comment by mylinuxforwork** [comment]
@PSGitHubUser1 Can we say that this is a local issue independent form the Dotfiles and close therefor this issue?

**Comment by PSGitHubUser1** [comment]
> @PSGitHubUser1 Can we say that this is a local issue independent form the Dotfiles and close therefor this issue?

I don't know, you can but it's an issue. Like how will work?

Also, a senseless point, is that it's happening on your dotfile after all. Just seeking help, @Ahorts got the same issue too. If you close this issue then it won't show up in the 'Issues' tab & nobody is coming to help. Although, it's your choice, if you want to close it, you can.

**Comment by mylinuxforwork** [comment]
@PSGitHubUser1 Can you open pavucontrol and check your audio settings. Just click in the volume module on Waybar.

**Comment by PSGitHubUser1** [comment]
> @PSGitHubUser1 Can you open pavucontrol and check your audio settings. Just click in the volume module on Waybar.

Yes, I tried pyvucontrol already, before even making this issue on GitHub.

**Comment by mylinuxforwork** [comment]
I mean this module:

![image](https://github.com/user-attachments/assets/6ae1aa14-d831-474f-9319-a6ef24aaef6a)

How does it look on your side?

**Comment by Ahorts** [comment]
When the speaker icon opens volume control, I go to the output devices and then mute and unmute my speaker before the audio works.
![screenshot_25072024_195543](https://github.com/user-attachments/assets/1c3865c3-bc78-4663-a924-44cb0d409d07)

**Comment by Ahorts** [comment]
@PSGitHubUser1 search volume control in rock and see what comes up

**Comment by PSGitHubUser1** [comment]
> @PSGitHubUser1 search volume control in rock and see what comes up

Finally, it worked. I reinstalled the dotfile, had to waste a lot of time but okay.

**Comment by mylinuxforwork** [comment]
Great.
